bundles/org.openhab.binding.tacmi/README.md

```diff
@@ -8,25 +8,35 @@ These modes are:
 
 
 Via a "Schema API Page"
+
   * Read values from output nodes
   * Change values for controllable nodes
 
 CoE (CAN over Ethernet) Connection
+
   * Receive data from analog CAN-outputs defined in TAPPS2
   * Receive data from digital CAN-outputs defined in TAPPS2
   * Send ON/OFF to digital CAN-inputs defined in TAPPS2
   * Send numeric values to analog CAN-inputs defined in TAPPS2
+  
+JSON API
 
+  * Read selected data via the JSON api.
+    See [API documentation](https://wiki.ta.co.at/C.M.I._JSON-API) for details
 
-Depending on what you want to achieve, either the "Schema API Page" or the CoE way might be better.
-As rough guidance: Anything you want to provide to the TA equipment it has to work / operate with the CoE might be better.
+Depending on what you want to achieve, either the "Schema API Page", the CoE or the JSON API way might be better.
+As rough guidance:
+If you just want to collect values the JSON API is probably the way to go.
```

Comment:
I changed the default to La,Ld. That's the data that is logged.

Reply:
Does not help for my UVR1611, only supported by X2 according to the documentation. But as the user needs to read the JSON doc anyway, any default is fine for me.

Reply:
Did you already push your changes?

Reply:
Ah right, I have a X2 (I'm not programming it my self, but it came with my heat pump). Should I keep it with "I,O,Sg"?

Talking about 1611 vs. 16x2: I mainly implemented the Json-Stuff, because the "Schema API" didn't work for me. 

I just pushed it. I'm working on an much to old notebook for java-compilation (verify took forever). ;)

Reply:
i,O, Sg might be better as starting point, as it gives results for most/all UVR devices.

Reply:
Ok. Reverted back to I,O,Sg. I'm also unhappy with the term "Possible options are", because it could suggest, that there are no others. But as you said: The API-Docs need to be read anyway. :-)